proto/stride/records/records.proto

```diff
@@ -9,7 +9,7 @@ option go_package = "github.com/Stride-Labs/stride/v16/x/records/types";
 message UserRedemptionRecord {
   string id = 1; // {chain_id}.{epoch}.{receiver}
   string receiver = 3;
-  string amount = 4 [
+  string native_token_amount = 4 [
```

Comment:
Can we rename a field like this? It's possible we need to depracate the field and create a new one with a new index, I'm not sure. I bet @sampocs will know.

Reply:
no need to deprecate or create a new index, just a renaming is fine!

Reply:
It's a good question though! We should definitely double check this in localstride (along with all of the unbonding records)

Reply:
For sure! Fwiw, we did this on our host zone struct in the lsm upgrade - renamed all ica address fields and did not do a migration

Reply:
Great. Resolving.